ui/src/app/machines/views/MachineList/MachineListHeader/ActionFormWrapper/DeployForm/DeployFormFields/UserDataField/UserDataField.test.tsx

```diff
@@ -0,0 +1,244 @@
+import { act } from "react-dom/test-utils";
+import configureStore from "redux-mock-store";
+import { mount, ReactWrapper } from "enzyme";
+import { MemoryRouter } from "react-router-dom";
+import React from "react";
+import { Provider } from "react-redux";
+
+import DeployForm from "../../DeployForm";
+import { TSFixMe } from "app/base/types";
```

Comment:
We should be using [type only imports](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export), so this should be `import type { TSFixMe } from "app/base/types";`

Reply:
Oh nice, I didn't know about that, I'll add those when I move the store around.